client/state/plugins/wporg/selectors.js

```diff
@@ -1,25 +1,22 @@
-export const getPlugin = function ( state, pluginSlug ) {
-	if ( ! state || ! state[ pluginSlug ] ) {
-		return null;
-	}
-	return Object.assign( {}, state[ pluginSlug ] );
-};
+export function getAllPlugins( state ) {
+	return state?.plugins.wporg.items;
```

Comment:
Judging by how the plugins reducer tree is loaded, it seems it'll always be there, making the `state?.`  unnecessary. However, I realize it's needed if we want to keep the previous logic, so I'm fine with keeping it as-is.

Reply:
It's been a while, but I believe the main reason for keeping it like this was that some unit tests started failing, because they expect to be able to set the whole of state to `null` or `undefined`.